bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -110,7 +110,7 @@ $bars-padding: $lg-padding-x - .45rem; // -.45 so user-list and chat title is al
   }
 
   @include mq($xlarge-up) {
-    flex-basis: 10vw;
+    flex-basis: 13vw;
```

Comment:
This isn't changing the default size of the userlist or chat.

Reply:
I have just changed the default size of $xlarge-up, thanks.